### ISSUE TITLE
MarkdownShare.com not working

### ISSUE THREAD

**Issue by MoaufmKlo** [skx/markdownshare#4]
I get redirected to this when trying to create a markdown resource:

> There was an error creating/saving your markdown resource:
> 
> open store/0/b/8/1/0b8158ec-c8f1-4c21-87cb-0da70d691838.TEXT: permission denied

**Comment by skx** [comment]
Yeah I've been a bit busy recently; I migrated the service to a new host, imported all the data but didn't setup the users properly so writing new entries, editing existing ones, or deleting them fails.

I expected that I'd be letting things degrade gracefully in the near future - hence #3 - but it seems like the migration broke it enough already that it works.

My plan is:

* Fix the deployment such that it works.
  * Fixing it in-place is trivial, but I want to be able to migrate in an automated fashion and that's complicated for internal-reasons.
  * (I script deployments, along with restoration, but testing the process requires spinning up a few VMs, deploying, migrating, comparing state, and shutting down, so it takes a few hours from start to finish.)
* Then add a banner to all pages saying "Going read-only from June 1st 2020", or similar.
* Then disable new pages/updates.

The service is basically an abuse magnet, and I don't have the time/patience to take care of it.

I'll aim to get this working by Tomorrow.  Shouldn't be hard, but it's been at the bottom of my todo-list since it was moved on [28th January](https://blog.steve.fi/initial_server_migration_complete__.html) as I didn't receive a single complaint until your issue!
